jerry-core/api/jerry-snapshot.c

```diff
@@ -494,24 +495,26 @@ snapshot_load_compiled_code (const uint8_t *base_addr_p, /**< base address of th
 
 #endif /* JERRY_ENABLE_SNAPSHOT_EXEC */
 
+#ifdef JERRY_ENABLE_SNAPSHOT_SAVE
```

Comment:
I think we should do the same here as in other API functions. Move the guard into the function's body and use `JERRY_UNUSED`, if snapshot save is disabled. We usually prefer no-op rather than a compile time liner error.

Reply:
Oh, my bad. It is only a static function.